handlers/search.go

```diff
@@ -267,7 +265,42 @@ func SearchAhead(w http.ResponseWriter, r *http.Request) {
 				WHERE from_address_text LIKE $1 || '%'
 			) a 
 			GROUP BY from_address_text`, lowerStrippedSearch)
-
+	case "count_indexed_validators_by_withdrawal_credential":
+		var ensData *types.EnsDomainResponse
+		if utils.IsValidEnsDomain(search) {
+			ensData, _ = GetEnsDomain(search)
+			if len(ensData.Address) > 0 {
+				lowerStrippedSearch = strings.ToLower(strings.Replace(ensData.Address, "0x", "", -1))
+			}
+		}
+		if !searchLikeRE.MatchString(lowerStrippedSearch) {
+			break
+		}
+		const beginningOfSetCredentials string = "010000000000000000000000"
+		if len(lowerStrippedSearch) <= 40 && lowerStrippedSearch[:24] != beginningOfSetCredentials {
+			lowerStrippedSearch = beginningOfSetCredentials + lowerStrippedSearch
+		}
+		// find validators per withdrawal credential (limit result by N addresses and M validators per credential)
```

Comment:
**question:** I see that a lot of the search options have this kind of comment but is there a limit?

Reply:
Indeed, case `count_indexed_validators_by_eth1_address` does not implement any limit, neither do I.
I removed these comments.